.nvmrc

```diff
@@ -0,0 +1 @@
+lts/gallium
```

Comment:
Same

Reply:
needed to enforce version. It is not a good idea to let the version not set. This is a lts btw